frontend/src/components/board/BoardCreationForm.tsx

```diff
@@ -55,8 +83,27 @@ const BoardCreationForm: React.FC<AddBoardCreationFormProps> = ({ onSubmit, onCa
             })}
           />
         </Grid>
-        <Grid item xs={12}>
-          <TextField label="Password" type="password" {...register("password")} />
+        <Grid item xs={9.75}>
```

Comment:
Näyttää ainakin mun mielestä oudolta, jos password field eri kokonen kun board name:n field. 

```suggestion
        <Grid item xs={11}>
```

Reply:
Komponentti on skaalattu pienemmäksi, jotta se automaattisesti sopii yhteen vanhemman koodin kanssa visuaalisesti. Itemien koko on nyt siis sama.

Reply:
![image](https://github.com/user-attachments/assets/638dfe0a-511e-4632-ba82-7f88f19cea49)


Mulla nämä kaksi on oikeastaan edelleen eri kokoiset.

Lisäisin esim. `TextField` componenttiin (BoardCreationForm & BoardImportForm) `sx={{width: 80%}}`, niin olisi molemmat samankokoisia. Oikeastaan mun silmäään parhaalta näyttää, että molemmat olisi 90% (siis myös `PasswordField`, mutta se on makuasia).

Reply:
![Futuboard-form-bug-screenie-01](https://github.com/user-attachments/assets/c5bb2aef-238e-4364-ad51-92e553dc5c1f)

Outoa, selvitän asian. Kiitos.

Reply:
Päivitin ehdotetut sx-arvot ko. komponentteihin. Nyt kentät ovat samankokoisia Chromessa ja Firefoxissa ainakin minun puolella.